plugins/ingest-attachment/build.gradle

```diff
@@ -71,9 +71,9 @@ dependencies {
   api "org.apache.pdfbox:fontbox:${versions.pdfbox}"
   api "org.apache.pdfbox:jempbox:1.8.16"
   api "commons-logging:commons-logging:${versions.commonslogging}"
-  api "org.bouncycastle:bcmail-jdk15to18:${versions.bouncycastle}"
-  api "org.bouncycastle:bcprov-jdk15to18:${versions.bouncycastle}"
-  api "org.bouncycastle:bcpkix-jdk15to18:${versions.bouncycastle}"
+  api "org.bouncycastle:bcmail-jdk18on:${versions.bouncycastle}"
```

Comment:
```suggestion
  api "org.bouncycastle:bcmail-jdk15to18:${versions.bouncycastle}"
```

Reply:
(and for others)

Reply:
Same as https://github.com/opensearch-project/OpenSearch/pull/13484/files#r1586969782